### PR TITLE
Compress the string

### PR DESCRIPTION
**Pull request by Vaibhavabhaysharma**
Through this code I would like  to appreciate the usefulness of the groupby() function of itertools for compressing the string and printing it with its number of occurences.

### CODE REVIEW

File: Basic-Scripts/Compress_the_string.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from itertools import groupby
+S=str(input())                                          # user input for string
```
Comment (SKAUL05) [inline Basic-Scripts/Compress_the_string.py]:
```suggestion
uncompressed = str(input("Enter string to compress: "))                                          # user input for string
```

File: Basic-Scripts/Compress_the_string.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+print(*[(len(list(c)), str(k)) for k, c in groupby(S)]) # logic to print occurence of values in string
```
Comment (SKAUL05) [inline Basic-Scripts/Compress_the_string.py]:
```suggestion
print(*[(len(list(value)), str(key)) for key, value in groupby(uncompressed)]) # logic to print occurence of values in string 
```

**Review by SKAUL05 — CHANGES_REQUESTED** [review]

**Review by SKAUL05 — CHANGES_REQUESTED** [review]
Change filename to `compress_the_string.py`

**Review by Vaibhavabhaysharma — COMMENTED** [review]
Resolved requested issue...

**Review by SKAUL05 — APPROVED** [review]

**Comment by SKAUL05** [comment]
Good to go @ankitdobhal

**Pull request merged**